Classes/Hooks/T3libTcemain.php

```diff
@@ -45,7 +45,7 @@ class T3libTcemain {
 	 */
 	public function checkRecordUpdateAccess($table, $id, $data, $res, $tcemain) {
 		/** @var tx_languagevisibility_beservices $visibilityservice */
-		$visibilityservice = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('tx_languagevisibility_beservices');
+		$visibilityservice = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(BeServices::class);
```

Comment:
This will not work as it's still php 5.3 compatible, and as long as the extension is TYPO3 6.2 compatible, it has to stay 5.3 compatible too.